### PR TITLE
dx-1207 changes to config with new kramdown

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -9,15 +9,18 @@ plugins:
...
+    syntax_highlighter_opts:
+      guess_lang: true
+      default_lang: html
+      css_class: "code-view"
+      code_class: "code-view-code"
+      table_class: "code-view-table"
+      gutter_class: "code-view-numbers"
+      block:
+        line_numbers: true
+        start_line: 1
```
Comment (asbjornu) [inline _config.yml:23]: These should all be less indented.

```suggestion
  input: GFM
  syntax_highlighter: rouge
  syntax_highlighter_opts:
    guess_lang: true
    default_lang: html
    css_class: "code-view"
    code_class: "code-view-code"
    table_class: "code-view-table"
    gutter_class: "code-view-numbers"
    block:
      line_numbers: true
      start_line: 1
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
You seem to be missing quite a few `code-header` replacements?

File: index.md [hunk 2]
```diff
@@ -2,6 +2,157 @@
 title: Developer Portal
 front_page: true
```
Comment (asbjornu) [inline index.md:3]:
```suggestion
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398&bid=20178598)

File: _includes/moto.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+{% include alert-agreement-required.md %}
+
+Mail-Order Telephone-Order
+Notes to ourself: 
+  * only supported by Purchase (not Verify), 
+  * Only valid with 3DS 2.x
+  * skips SCA
+  * Should be Type = MOTO and acquirerTransactionType = MOTO on the response for GET payment under payment/transactions/{authorize}/activities/{authorize}
+
+{:.code-view-header}
```
Comment (payex-dev) [inline _includes/moto.md:12]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)

File: _includes/moto.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+**Request**
+
```
Comment (payex-dev) [inline _includes/moto.md:14]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)

File: _includes/moto.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+```http
```
Comment (payex-dev) [inline _includes/moto.md:15]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)

File: _includes/moto.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
+## MOTO
+
```
Comment (payex-dev) [inline _includes/moto.md:2]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)

File: _includes/moto.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+Mail-Order Telephone-Order
+Notes to ourself: 
+  * only supported by Purchase (not Verify), 
+  * Only valid with 3DS 2.x
+  * skips SCA
+  * Should be Type = MOTO and acquirerTransactionType = MOTO on the response for GET payment under payment/transactions/{authorize}/activities/{authorize}
+
+{:.code-view-header}
+**Request**
```
Comment (payex-dev) [inline _includes/moto.md:13]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398&bid=20178598)

File: _includes/moto.md (hunk 4 above)
Comment (payex-dev) [inline _includes/moto.md:14]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)

File: _includes/moto.md (hunk 7 above)
Comment (payex-dev) [inline _includes/moto.md:13]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)

File: _includes/moto.md (hunk 3 above)
Comment (payex-dev) [inline _includes/moto.md:12]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)

File: _includes/moto.md (hunk 6 above)
Comment (payex-dev) [inline _includes/moto.md:2]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)

File: _includes/moto.md (hunk 5 above)
Comment (payex-dev) [inline _includes/moto.md:15]: Codacy found an issue: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6182398)

File: _includes/moto.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
+## MOTO
```
Comment (asbjornu) [inline _includes/moto.md:1]: Why is `moto.md` added in this PR? Doesn't it belong to #671?

**Review by asbjornu — CHANGES_REQUESTED** [review]
Shouldn't theme be upgraded to version 1.8 in this PR as well? Or has that been done in a previous PR already?

**Review by NullableInt — APPROVED** [review]
LGTM

**Review by asbjornu — CHANGES_REQUESTED** [review]
Why isn't theme upgraded to version 1.8 like in #680?

File: index.md [hunk 9]
```diff
@@ -1,5 +1,157 @@
 ---
 title: Developer Portal
-layout:front_page
+layout: front_page
```
Comment (asbjornu) [inline index.md:3]: The name of the layout is `front-page`.

```suggestion
layout: front-page
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by asbjornu — APPROVED** [review]
🎉

**Pull request merged**